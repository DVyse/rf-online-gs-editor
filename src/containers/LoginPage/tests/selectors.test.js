import { fromJS } from 'immutable';
import makeSelectLoginPage, { selectLoginPageDomain } from '../selectors';

describe('selectLoginPageDomain', () => {
  it('returns the initial state', () => {
    expect(selectLoginPageDomain(fromJS({}))).toMatchSnapshot();
  });
  it('returns the loginPage state', () => {
    expect(
      selectLoginPageDomain(
        fromJS({
          loginPage: { test: 1, value: 2 },
        }),
      ),
    ).toMatchSnapshot();
  });
  it('returns the loginPage state object', () => {
    expect(
      makeSelectLoginPage()(
        fromJS({
          loginPage: { test: 1, value: 2 },
        }),
      ),
    ).toMatchSnapshot();
  });
});
