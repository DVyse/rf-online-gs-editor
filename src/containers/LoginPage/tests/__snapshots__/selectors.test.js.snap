// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selectLoginPageDomain returns the initial state 1`] = `
Immutable.Map {
  "ident": "",
  "password": "",
  "isLoading": false,
  "isError": false,
  "errorMessage": "",
}
`;

exports[`selectLoginPageDomain returns the loginPage state 1`] = `
Immutable.Map {
  "test": 1,
  "value": 2,
}
`;

exports[`selectLoginPageDomain returns the loginPage state object 1`] = `
Object {
  "test": 1,
  "value": 2,
}
`;
